ext/node/ops/vm.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::error::type_error;
+use deno_core::error::AnyError;
+use deno_core::op2;
+use deno_core::v8;
+
+use super::vm_internal as i;
+
+struct Script {
+  inner: i::ContextifyScript,
+}
+
+impl Script {
+  fn new(
+    scope: &mut v8::HandleScope,
+    source: v8::Local<v8::String>,
+  ) -> Result<Self, AnyError> {
+    Ok(Self {
+      inner: i::ContextifyScript::new(scope, source)?,
+    })
+  }
+
+  fn run_in_this_context<'s>(
+    &self,
+    scope: &'s mut v8::HandleScope,
+  ) -> Result<v8::Local<'s, v8::Value>, AnyError> {
```

Comment:
```suggestion
  #[op2(method)]
  fn run_in_this_context<'s>(
    &self,
    scope: &'s mut v8::HandleScope,
  ) -> Result<v8::Local<'s, v8::Value>, AnyError> {
```

This is designed to be easily ported to op2++ when it lands (soon™)